### PR TITLE
Auto Shoot uses SequentialCommandGroups and is triggered by hold press

### PR DESCRIPTION
**Pull request by Calculus-Master**

### CODE REVIEW

File: RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java [hunk 1]
```diff
@@ -34,27 +34,81 @@ public AutoShootCommand withParameters(int ballCount, boolean top)
...
-                                .andThen(this::disableShooterFeedersBMS)));
+        SequentialCommandGroup command = new SequentialCommandGroup();
+
+        command.addCommands(
+
+                //Activate shooter
+                new InstantCommand(this.top ? () -> this.shooter.spinUpTop() : () -> this.shooter.shootLow()),
+
+                //Wait for shooter to get up to speed
+                new WaitCommand(0.3), //new WaitUntilCommand(() -> this.shooter.isUpToSpeed())
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java:45]: we should use isUpToSpeed here.

Comment (Calculus-Master) [reply RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java:45]: Oh yeah it should be more accurate now right

Comment (cpostbitbuckets) [reply RapidReact/src/main/java/frc/robot/commands/AutoShootCommand.java:45]: yep!

**Review by cpostbitbuckets — APPROVED** [review]

**Comment by Calculus-Master** [comment]
Also I'll undraft this after we test it at XC
Just in case the WaitCommands don't actually wait or something

**Review by cpostbitbuckets — APPROVED** [review]

**Pull request merged**